### PR TITLE
Consolidation of LlamaIndex and LangChain Dependency Versions

### PR DESCRIPTION
**Pull request by MinuraPunchihewa**
## Description

This PR consolidates the dependency versions for the LlamaIndex and LangChain integrations.

Once this is approved, I will also raise another PR to update these versions in Cloud.

Fixes https://github.com/mindsdb/mindsdb/issues/8252

## Type of change

(Please delete options that are not relevant)

- [X] 🐛 Bug fix (non-breaking change which fixes an issue)

## Verification Process

To ensure the changes are working as expected:

 - [ ]   Tested locally after updating versions of the dependencies given

## Additional Media:

- [X] I have attached a brief loom video or screenshots showcasing the new functionality or change.

Given below is a screenshot of running the integration after installing the new versions of the depdencies,
image

Given below are the versions of `llama-index` and `langchain` that have been used for the above,
image

## Checklist:

- [X] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have appropriately commented on my code, especially in complex areas.
- [ ] Necessary documentation updates are either made or tracked in issues.
- [ ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

**Review by ZoranPandovski — APPROVED** [review]
Lgtm, everything works. Waiting on @paxcema approval. Once merged we ca solve it on cloud

**Review by paxcema — APPROVED** [review]

**Pull request merged**
